ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -643,6 +649,30 @@ tests:
     env:
       ALLOW_CONVERGED_FLOW: "True"
     workflow: assisted-baremetal-operator-ztp
+- as: e2e-ai-operator-ztp-multiarch-ocp-411
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate-4-11-multi
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
```

Comment:
just to make sure (because maybe as-is is the intended configuration)
we want the spoke to use multiarch image and the hub-cluster to be based on x86?

Reply:
Yes, exactly that. I want a regular x86 Hub (i.e. _I don't care, I just want a Hub_) but the spoke should be using multiarch image